app/components/listing/__tests__/RelativeOpeningTimes.spec.jsx

```diff
@@ -33,32 +34,34 @@ describe('<RelativeOpeningTimes />', () => {
   });
 
   it('should display "Closed until tomorrow" for a schedule that is closed until tomorrow', () => {
-    const date = new Date();
-    date.setHours(18);
-    date.setMinutes(15);
+    const date = moment().startOf('day').add(18, 'hours').add(15, 'minutes');
     const schedule = getNamedSchedule('nine_to_six');
     const comp = shallow(<RelativeOpeningTime schedule={schedule} currentDate={date} />);
     expect(comp.find('.relative-opening-time').text()).to.equal('Closed Until Tomorrow');
     expect(comp.find('.relative-opening-time').hasClass('status-red')).to.be.true;
   })
 
   it('should display "Opens in 5 minutes" for a schedule that is just about to open', () => {
-    const date = new Date();
-    date.setHours(8);
-    date.setMinutes(55);
+    const date = moment().startOf('day').add(8, 'hours').add(55, 'minutes');
     const schedule = getNamedSchedule('nine_to_six');
     const comp = shallow(<RelativeOpeningTime schedule={schedule} currentDate={date} />);
     expect(comp.find('.relative-opening-time').text()).to.equal('Opens in 5 mins');
     expect(comp.find('.relative-opening-time').hasClass('status-amber')).to.be.true;
   });
 
   it('should display "Closes in 10 minutes" for a schedule that is just about to close', () => {
-    const date = new Date();
-    date.setHours(17);
-    date.setMinutes(50);
+    const date = moment().startOf('day').add(17, 'hours').add(50, 'minutes');
     const schedule = getNamedSchedule('nine_to_six');
     const comp = shallow(<RelativeOpeningTime schedule={schedule} currentDate={date} />);
     expect(comp.find('.relative-opening-time').text()).to.equal('Closes in 10 mins');
     expect(comp.find('.relative-opening-time').hasClass('status-amber')).to.be.true;
   });
+
+  it('should display "Closed until tomorrow" for a schedule that was open today, but has closed until tomorrow', () => {
+    const date = moment().startOf('day').add(18, 'hours').add(30, 'minutes');
+    const schedule = getNamedSchedule('nine_to_six');
```

Comment:
Is this the right schedule? It looks like this is a normal 9am-6pm schedule.

Reply:
Yes, because the test feeds in a fake current time as 6:30pm, so it was open today, closed 30m ago and isn't open until 9am tomorrow

Reply:
Ah, I realized I completely misread the description of the test. I mistakenly thought that this was going to be about the case where you have a close time that is before the open time, indicating that the close time should actually be one day in the future.